visualisation-server/rockcraft.yaml

```diff
@@ -55,7 +56,7 @@ parts:
   
   non-root-user:
     plugin: nil
-    after: [mlflow]
+    after: [python, gcloud, files]
```

Comment:
Do we really need this?
I thought `non-root-user` part does not depend on those.